A/AFLplusplus/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+using BinaryBuilder, Pkg.Types
+
+name = "AFLplusplus"
+version = v"2.68"
+sources = [
+    ArchiveSource("https://github.com/AFLplusplus/AFLplusplus/archive/2.68c.tar.gz", "862e155c97737770baa26ffedf324a7fa255b757c85b0c9a6f312264f2ca29c5"),
+    ArchiveSource("http://download.qemu-project.org/qemu-3.1.0.tar.xz", "6a0508df079a0a33c2487ca936a56c12122f105b8a96a44374704bef6c69abfc", unpack_target="AFLplusplus-2.68c/qemu_mode"),
```

Comment:
Why not using [Qemu](https://github.com/JuliaPackaging/Yggdrasil/blob/ebf6754c1f3a95751cfe3727f9224d0e80d3711b/Q/Qemu/build_tarballs.jl) as dependency?  It's better to be as modular as possible, instead of duplicating the effort

Reply:
The Qemu binary compiled in this package is heavily patched to include the instrumentation logic (patches are in the aflplusplus source)